code/modules/projectiles/gun_system.dm

```diff
@@ -1562,6 +1562,11 @@
 			iff_signal = sentry.iff_signal
 		projectile_to_fire.iff_signal = iff_signal
 	projectile_to_fire.damage_marine_falloff = iff_marine_damage_falloff
+	//no point blank bonus when akimbo
+	if(dual_wield == TRUE)
+		projectile_to_fire.point_blank_range = 0
+	else
+		projectile_to_fire.point_blank_range = projectile_to_fire.ammo.point_blank_range
```

Comment:
I dont think I like how this is outside the gun code 
Stuff like UAVS and etc that use gun code will not respect it anymore

Reply:
updated